lib/routes.php

```diff
@@ -356,7 +356,6 @@
  * )
  */
 $app->get('/stats', ['\Xibo\Controller\Stats','grid'])->setName('stats.search');
-$app->get('/stats/timeDisconnected', ['\Xibo\Controller\Stats', 'gridTimeDisconnected'])->setName('stats.timeDisconnected.search');
```

Comment:
This is an API which we put in for a user request

Reply:
If I put this back, I think that is fine. We are not using this method in any of our reports